### DISCUSSION TITLE
2.x missing Wifi.h

### DISCUSSION THREAD

**Discussion by Tintai** [luc-github/ESP3D#1017] [Questions]
Hello, I'm doing clean build of latest version (2.1) and getting error for missing WiFi.h which is not needed for 8266 build. Version 3 alpha is building without problems but can't compile 2.x for 8266 (32 is compiling well)

![error](https://github.com/luc-github/ESP3D/assets/16637080/c65339ff-fd65-4f96-87fc-cc4322864d4a)

Thanks

**Comment by luc-github** [comment]
what version are you compiling ?

**Reply by Tintai** [reply]
Any 2.x version is not compiling for me. This one is the latest.

**Comment by luc-github** [comment]
I cannot duplicate with latest version  / git
![image](https://github.com/luc-github/ESP3D/assets/8822552/9e51747a-d6dc-498c-b7ff-893edd1bd977)

**Reply by Tintai** [reply]
I've tried once again and It worked for 2.1.4.

But for 2.1.3b0 still getting this file missing.
![abcsss](https://github.com/luc-github/ESP3D/assets/16637080/a019a9ff-3bdb-46fa-a650-9072d6767804)


Anyway latest seems to work fine, but I would like to know whats about that error

**Reply by luc-github** [reply]
read commit history https://github.com/luc-github/ESP3D/commit/8cff8c44fbbb3ac7747a6ab4a981147c9102866a
